include/mpsl/mpsl_cx_software.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/**
+ * @file mpsl_cx_software.h
+ *
+ * @defgroup mpsl_cx_software Multiprotocol Service Layer Software Coexistence
+ *
+ * @brief MPSL Software Coexistence
+ * @{
+ */
+
+#ifndef MPSL_CX_SOFTWARE__
+#define MPSL_CX_SOFTWARE__
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <zephyr/sys_clock.h>
+
+#include <stdint.h>
+
+#ifdef CONFIG_MPSL_CX_SOFTWARE_RPC_CLIENT
+/*
+ * MPSL Software Coexistence RPC client provides serialization of the MPSL Software Coexistence
+ * APIs, but it does not pull the actual MPSL Radio Coexistence library into the build system.
+ * Hence the 'mpsl_cx_op_map_t' must be redefined below.
+ */
+typedef uint8_t mpsl_cx_op_map_t;
```

Comment:
I think you should provide `mpsl_cx_op_t` type from
https://github.com/nrfconnect/sdk-nrfxlib/blob/main/mpsl/include/protocol/mpsl_cx_protocol_api.h#L43

along with `mpsl_cx_op_map_t`. Otherwise we won't have values to build `ops` parameter of `mpsl_cx_software_set_granted_ops`